src/pages/Home/Home.page.js

```diff
@@ -0,0 +1,69 @@
+import React, { useRef, useContext } from 'react'
+import { Context } from '../../context/context'
+import Card from '../../components/Card/Card'
+// import { Link, useHistory } from 'react-router-dom'
```

Comment:
Do not keep commented out code, remove it. You have Git history in case you need these back.